pyodide-build/pyodide_build/cli/build.py

```diff
@@ -127,6 +112,10 @@ def main(
         "",
         help="Build source, can be source folder, pypi version specification, or url to a source dist archive or wheel file. If this is blank, it will build the current directory.",
     ),
+    output_directory: str = typer.Option(
```

Comment:
```suggestion
    output_directory: Path = typer.Option(
```

Reply:
Okay, so it's relying on some decorator magic. Slightly concerning but okay.

Reply:
So if you say:
```
a : A = typer.Option("blah")
```
does it try to set `a = A("blah")`? Or is it using a table of known conversions or something?

Reply:
Looking at https://github.com/tiangolo/typer/issues/77 looks like there is a pre-defined list of types. But Path and dates are the most useful.

Reply:
Yes, there are some [pre-defined list of types](https://typer.tiangolo.com/tutorial/parameter-types/path/) that typer can handle. I mean, it is not critical so we can just use the type `str` and handle it manually if you feel concerned or if we don't want to rely too much on internal features of typer or click.

Reply:
No I'm good with it thanks for the suggestion and for explaining!